Python_Scripts/script_download_all_sbml_biomodels.py

```diff
@@ -26,33 +28,21 @@
 # all unique model identifications consisting of pairs of id and name
 #  Remark: all ids at BioModels are unique and don't change over time
 models = pd.DataFrame(columns=['model_id', 'model_name'], data=[])
-model_ids = [
-    'BIOMD0000000334', 'BIOMD0000000332', 'BIOMD0000000333', 'BIOMD0000000451',
-    'BIOMD0000000161', 'BIOMD0000000070', 'BIOMD0000000560', 'BIOMD0000000574',
-    'BIOMD0000000583', 'BIOMD0000000021', 'BIOMD0000000014', 'BIOMD0000000303',
-    'BIOMD0000000568', 'BIOMD0000000611', 'BIOMD0000000147', 'BIOMD0000000559',
-    'BIOMD0000000491', 'BIOMD0000000492', 'BIOMD0000000172', 'BIOMD0000000286',
-    'BIOMD0000000293', 'BIOMD0000000488', 'BIOMD0000000504', 'BIOMD0000000634',
-    'BIOMD0000000543', 'BIOMD0000000544', 'BIOMD0000000049', 'BIOMD0000000579',
-    'BIOMD0000000151', 'BIOMD0000000397', 'BIOMD0000000469', 'BIOMD0000000472',
-    'BIOMD0000000022', 'BIOMD0000000205', 'BIOMD0000000106', 'BIOMD0000000505']
-
-model_names = [
-    'Bungay2003', 'Bungay2006', 'Bungay2006a', 'Carbo2013',
-    'Eungdamrong2007', 'Holzhutter2004', 'Hui2014', 'Lai2014',
-    'Leber2015', 'Leloup1999', 'Levchenko2000a', 'Liu2011',
-    'Mueller2015', 'Nayak2015', 'ODea2007', 'Ouzounoglou2014',
-    'Pathak2013', 'Pathak2013a', 'Pritchard2002', 'Proctor2010',
-    'Proctor2010a', 'Proctor2013', 'Proctor2013a', 'Proctor2013b',
-    'Qi2013', 'Qi2013a', 'Sasagawa2005', 'Sengupta2015',
-    'Singh2006', 'Sivakumar2011c', 'Smallbone2013d', 'Smallbone2013g',
-    'Ueda2001', 'Ung2008', 'Yang2007', 'vanEunen2013']
+
+with open(BIOMODELS_MODEL_SLUG_FILE) as f:
+    model_ids = f.readlines()
+model_ids = [x.rstrip('\n') for x in model_ids]
+
+with open(BIOMODELS_MODEL_FILE) as f:
+    model_names = f.readlines()
+model_names = [x.rstrip('\n') for x in model_names]
```

Comment:
:+1:

Reply:
(note: was needed for enabling a smaller test set)